### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by John-athan14 — DISMISSED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ import './App.css';
 
 function App() {
   return (
-    <div className="homePage">
+    <div
+      className="homePage"
+      style={{ backgroundImage: 'url(assets/img/backmg.jpg)' }}
```
Comment (Hacene010) [inline src/App.jsx:13]: no style here. You can put it into the app.css

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Need to correct one thing

**Review by John-athan14 — DISMISSED** [review]

**Review by John-athan14 — DISMISSED** [review]

File: src/Containers/CardDescription.jsx [hunk 2]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
...
+          <p className="cardType">Rarity :&nbsp;{card.rarity}</p>
           {card.flavor_text ? (
-            <p className="cardFlavour"> {card.flavor_text}</p>
+            <p className="cardFlavour">{card.flavor_text}&#34;</p>
           ) : (
             ''
           )}
+          {card.power ? (
+            <p className="cardPower">
+              {' '}
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: - You can remove this

File: src/Containers/CardDescription.jsx [hunk 3]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
...
           <p className="cardOracle">{card.oracle_text}</p>
+          <p className="cardType">Set :&nbsp;{card.set_name}</p>
+          <p className="cardType">Rarity :&nbsp;{card.rarity}</p>
           {card.flavor_text ? (
-            <p className="cardFlavour"> {card.flavor_text}</p>
+            <p className="cardFlavour">{card.flavor_text}&#34;</p>
           ) : (
             ''
           )}
+          {card.power ? (
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: You can do it with this syntax : {card.power && ....}

File: src/Containers/CardDescription.jsx [hunk 4]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
...
+            <p className="cardPower">
+              {' '}
+              {card.power}/{card.toughness}
+            </p>
+          ) : null}
+          <p className="cardArtist">
+            🎨 &nbsp;&nbsp;{card.artist}&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;{' '}
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: - remove the {'  '} plz.

- code not very clean with these spaces

File: src/Containers/CardDescription.jsx [hunk 5]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
...
+          <p className="cardName">
+            {card.name}&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            {card.mana_cost}
+          </p>
           <p className="cardType">{card.type_line}</p>
+
           <p className="cardOracle">{card.oracle_text}</p>
+          <p className="cardType">Set :&nbsp;{card.set_name}</p>
+          <p className="cardType">Rarity :&nbsp;{card.rarity}</p>
           {card.flavor_text ? (
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: You can do it with this syntax : {card.flavor_text && ....}

File: src/Containers/CardDescription.jsx [hunk 6]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
       />
       <div className="cardAttribute">
         <div className="textContainer">
-          <p className="cardName">{card.name}</p>
+          <p className="cardName">
+            {card.name}&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: I think you can do better than 5 spacements like that

File: src/Containers/CardDescription.jsx [hunk 7]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
...
+            {card.mana_cost}
+          </p>
           <p className="cardType">{card.type_line}</p>
+
           <p className="cardOracle">{card.oracle_text}</p>
+          <p className="cardType">Set :&nbsp;{card.set_name}</p>
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: same here

File: src/Containers/CardDescription.jsx [hunk 8]
```diff
@@ -141,14 +141,30 @@ export default function CardDescription() {
...
+          <p className="cardType">Rarity :&nbsp;{card.rarity}</p>
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: same here.

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Saw some evils please correct them.

**Review by Hacene010 — APPROVED** [review]
Seems ok

**Review by John-athan14 — APPROVED** [review]

**Pull request merged**